precisely/comparison_matchers.py

```diff
@@ -45,3 +45,22 @@ def match(self, actual):
 
     def describe(self):
         return "{0} {1!r}".format(self._operator_description, self._value)
+
+
+def close_to(value, delta):
+    return IsCloseToMatcher(value, delta)
+
+
+class IsCloseToMatcher(object):
+    def __init__(self, value, delta):
+        self._value = value
+        self._delta = delta
+
+    def match(self, actual):
+        if actual - self._delta <= self._value <= actual + self._delta:
```

Comment:
Hmm, `value - delta <= actual <= value + delta` seems clearer to me (in that that's how I'd explain what's going on to somebody if I was talking to them), but happy to persuaded that as-is is clearer (or that my suggestion is just plain wrong).